pmd-doc/src/main/java/net/sourceforge/pmd/docs/GenerateRuleDocsCmd.java

```diff
@@ -47,16 +47,17 @@ public static void main(String[] args) throws IOException {
         System.out.println("Generated docs in " + (System.currentTimeMillis() - start) + " ms");
     }
 
+    static final Pattern ADDITIONAL_RULESET_PATTERN = Pattern.compile("^.+" + Pattern.quote(File.separator) + "pmd-\\w+"
+            + Pattern.quote(IOUtil.normalizePath(File.separator + Paths.get("src", "main", "resources", "rulesets").toString()) + File.separator)
```

Comment:
The fix is, to have the trailing "File.seperator" outside IOUtil.normalizePath - as it would be normalized away.